Assets/OfficeDoor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+//Simple handler for office door box collider
+public class OfficeDoor : MonoBehaviour
+{
+    private void OnTriggerEnter(Collider other)
+    {
+        transform.parent.Translate(100, 0, 0);
```

Comment:
if you want this to be extensible you can pull the vector out into a public variable that you can set in the inspector on various sized doors

Reply:
I did think about that. I figured I would have to actually specify individual axis variables, but it makes sense that unity inspector can  handle a myriad of complex types.  Thanks!